src/safeds/data/tabular/plotting/_table_plotter.py

```diff
@@ -572,7 +572,7 @@ def moving_average_plot(
                 ylabel=y_name,
             )
             ax.legend()
-            if self._table.get_column(x_name).is_temporal:
+            if self._table.get_column(x_name).is_temporal and self._table.get_column(x_name).row_count < 9:
```

Comment:
What difference does the added part of this line do? If I try it without the row count I get the same result

Reply:
this is for rows way greater than 9, when we have 1000 rows it tries to set as many dates as possible for the dates and the plots get messie

Reply:
So the idea is to limit the labels to a maximum of 8?